ports/espressif/bindings/espidf/__init__.c

```diff
@@ -110,10 +110,10 @@ const mp_obj_type_t mp_type_espidf_IDFError = {
 };
 
 
-//| class MemoryError(MemoryError):
-//|     """Raised when an ESP IDF memory allocation fails."""
-//|     ...
-//|
+// class MemoryError(MemoryError):
```

Comment:
Why delete this? Does the second MemoryError need to be prefixed or the first renamed?

Reply:
I commented this as it as was causing docs build to fail while I was testing something else and forgot about it. Fixed now.